src/execute_files.py

```diff
@@ -32,7 +32,8 @@ def create_outname(outdir, infile):
 def execute(infile, outfile, extras):
   """Create the command-line for an execution."""
   runner = extras['runner']
-  basename = os.path.splitext(os.path.basename(runner))[0]
+  # Only strip Windows suffxes because wasm.opt has one
```

Comment:
This comment feels incomplete? It reads like "we're only stripping Windows suffixes because wasm.opt has a .exe suffix," which I think is the opposite of what you mean?